platform/core/src/classes/MetadataProvider.js

```diff
@@ -1,8 +1,6 @@
 import queryString from 'query-string';
 import dicomParser from 'dicom-parser';
 import getPixelSpacingInformation from '../utils/metadataProvider/getPixelSpacingInformation';
-import fetchPaletteColorLookupTableData from '../utils/metadataProvider/fetchPaletteColorLookupTableData';
```

Comment:
Should also remove fetchpaletteColor at some point, but for now, just remove the commented out calls.